depends/funcs.mk

```diff
@@ -30,6 +30,18 @@ define fetch_file
     rm -rf $$($(1)_download_dir) ))
 endef
 
+define generate_crate_checksum
+rm .gitignore && \
+echo "{\"files\":{`find . -type f | grep -v "$($(1)_file_name)" | grep -v .stamp_ | sed 's|^./||' | sort | xargs $(build_SHA256SUM) | awk -v OFS='":"' '{print $$$$2, $$$$1}' | sed 's|^|"|' | sed 's|$|"|' | tr '\n' ',' | sed 's|,$$$$||'`},\"package\":\"$($(1)_sha256_hash)\"}" > .cargo-checksum.json
```

Comment:
Oww my head. Sorry, the interaction of quoting levels and of shell vs make expansion here is just impossible to review. NACK until it is simplified somehow (maybe an intermediate variable for the result of the backticked expression?)

Reply:
It doesn't help that it's all on one line.

Reply:
Or maybe a separate shell script?